### PR TITLE
Release preparation: README.md, CHANGELOG.md and build, configuration files improvements

### PR DESCRIPTION
**Pull request by pgev**
- README.md
- CHANGELOG.md
- CODE_OF_CONDUCT.md
- LICENSE
- Git configuration files improvements (.gitattributes, .gitignore)
- Solidity configuration files improvements (.solcover.js, .soliumignore, .soliumrc.json)
- Build files improvements (.travis.yml)
- Package.json: the following scripts are added `npm run compile`, `npm run compile-all`, `npm run lint:js`, `npm run lint:js:fix`, `npm run lint:sol`, `npm run lint:sol:fix`, `npm run ganache-cli`, `npm run build-package`.
- truffle.js improvements
- NPM publishing support

Initial version of README. Configuration files, build files, package.json scripts, travis, truffle cleanup as a prep step for 0.10.0 release.

Changelog for 0.10.0 version.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,19 +1,39 @@
...
-
-# VS Code
-.vscode
 
 # Vagrant
 .vagrant/
 ubuntu-xenial-16.04-cloudimg-console.log
 
+# don't commit contract binaries in protocol
+# we do commit the contract binaries in platform
```
Comment (benjaminbollen) [inline .gitignore]: this comment seems outdated; we would commit the bin and abi the in `build` directory

Comment (benjaminbollen) [inline .gitignore]: this comment seems from outdated code? is it needed?

File: contracts/organization/Organization.sol [hunk 2]
```diff
@@ -35,12 +35,6 @@ import "./OrganizationInterface.sol";
  */
 contract Organization is OrganizationInterface {
 
-
-    /* Using */
-
-    using SafeMath for uint256;
```
Comment (benjaminbollen) [inline contracts/organization/Organization.sol]: we should then also remove L23?

File: contracts/token/TokenRules.sol [hunk 3]
```diff
@@ -37,11 +35,6 @@ import "../organization/Organized.sol";
  */
 contract TokenRules is Organized {
 
-     /* Usings */
-
-    using SafeMath for uint256;
```
Comment (benjaminbollen) [inline contracts/token/TokenRules.sol]: because we don't use any uint operations? seems fair

File: .gitignore (hunk 1 above)
Comment (pgev) [reply .gitignore]: Agree. Done.

File: contracts/organization/Organization.sol (hunk 2 above)
Comment (pgev) [reply contracts/organization/Organization.sol]: Exactly. Thx. Done.

File: contracts/token/TokenRules.sol (hunk 3 above)
Comment (pgev) [reply contracts/token/TokenRules.sol]: Yes.

**Pull request merged**